images/ruby/3.0.tf

```diff
@@ -0,0 +1,37 @@
+module "three-zero" {
+  source = "../../tflib/publisher"
+
+  target_repository = var.target_repository
+  config            = file("${path.module}/configs/3.0.apko.yaml")
+}
+
+module "three-zero-dev" {
+  source = "../../tflib/publisher"
+
+  target_repository = var.target_repository
+  # Make the dev variant an explicit extension of the
+  # locked original.
+  config = jsonencode(module.three-zero.config)
+  extra_packages = concat(module.dev.extra_packages, [
+    "build-base",
+    "ruby-3.0-dev",
+    "ruby3.0-bundler",
+  ])
```

Comment:
I made this a local `node_dev_packages`, that might be easier than copying this.

Reply:
The problem is the version numbers :-/